### ISSUE TITLE
[4.1] Documentation for new 4.1 features

### ISSUE THREAD

**Issue by ceford** [joomla/joomla-cms#37051]
### Steps to reproduce the issue

Sorry to put this here - I have been asleep.

With the announcement of the release of 4.1 Stable I have been looking for documentation. I can't find anything on the new Tasks Scheduler. On Child Templates all I know of is the Youtube Video. I can't find anything on the Accessibility Checker. Syntax Highlighting and Inline Help I can see.

I found the Scheduled Tasks configuration in the Global Configuration screen and it has a placeholder Help screen. I could revise that if I knew something about scheduled tasks.

Any information placed here could lead to some user documentation!

**Comment by brianteeman** [comment]
The accessibility checker has its own dedicated site with everything documented https://joomla-projects.github.io/joomla-a11y-checker/

I wrote something for myself on some of the other features you are welcome to copy from https://domorewithcore.com

**Comment by ceford** [comment]
Thanks @brianteeman - really good docs that would be hard to better. Pity there were no links from the Release Announcement and I don't think there are any links yet from the Docs site. 

It may be useful for this issue to remain open for a few days!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37051">issues.joomla.org/tracker/joomla-cms/37051</a>.</sub>

**Comment by brianteeman** [comment]
> Thanks @brianteeman - really good docs that would be hard to better. 

@ceford I did post some of the text here a few weeks ago and thats being used in the release post

> Pity there were no links from the Release Announcement

Only so much I can do. The marketing team had access to the site

**Comment by brianteeman** [comment]
Is there anything left to do here?

**Comment by ceford** [comment]
Yes - Scheduled Tasks global configuration Help is still a list of placeholder headings. There is user documentation on Child Templates. Not sure about other stuff. I am doing something else just now.

**Comment by brianteeman** [comment]
This should be closed. Its not the place to report/log missing documentation on the documentation site
